cognite/client/_api/datapoints.py

```diff
@@ -1537,9 +1537,7 @@ def _bin_datapoints(self, dps_object_list: list[dict[str, Any]]) -> list[list[di
         return binned_dps_object_list
 
     def _insert_datapoints_concurrently(self, dps_object_lists: list[list[dict[str, Any]]]) -> None:
-        tasks = []
-        for dps_object_list in dps_object_lists:
-            tasks.append((dps_object_list,))
+        tasks = [(dps_object_list,) for dps_object_list in dps_object_lists]
```

Comment:
Looks like `list(zip(dps_object_lists))` would work 😜 Not that I think you should change it.

Reply:
I could go for both, honestly. The reader knows that this is "type/variable/data massaging" in both I guess. 🤷